src/test/unit/math/prim/mat/fun/value_of_rec_test.cpp

```diff
@@ -26,11 +26,17 @@ TEST(MathMatrix,value_of_rec) {
   
   Eigen::Matrix<double,2,5> a; 
   fill(a_vals, a);
+  Eigen::Map< Eigen::Matrix<double,2,5> > am(a.data(), a.rows(), a.cols());
```

Comment:
thanks for adding these tests! can you split this test into its own test case? I'd prefer if it weren't interleaved into the existing test.